packages/kodmq/.eslintrc.cjs

```diff
@@ -1,8 +1,4 @@
 module.exports = {
   root: true,
   extends: ["kodmq"],
-
-  rules: {
-    "import/extensions": ["error", "always"],
-  },
```

Comment:
No longer needed - can be enabled if this a personal preference of yours, my preference would be to not have the file extensions.

Reply:
Nice! Thanks